Test/WebTestCase.php

```diff
@@ -346,9 +353,51 @@ protected function postFixtureSetup()
     }
 
     /**
-     * Callback function to be executed after Schema restore.
+     * Callback function to be executed after Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function postFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function preFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed after save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
+     */
+    protected function postReferenceSave(ObjectManager $manager, AbstractExecutor $executor, $backupFilePath)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
      */
-    protected function postFixtureRestore()
```

Comment:
I do not know, as you may have noticed this Bundle is not necessarily on the top of my agenda right now. The above PR is already quite far along. So if you have some time to work on it .. it could likely get merged soon.